### PR TITLE
Replaces var edited mass driver machinery with subtypes

### LINKED ISSUE #54444: Pubby Toxins Mass Driver Pod Door No Toggle
I'm way too tired to look into this proper because mass drivers are a pain (to me), and just wanted to log it.

pub md

### PR DESCRIPTION
**Pull request by 81Denton**
## About The Pull Request
This PR replaces var edited mass driver machinery with proper subtypes.
It also fixes the Pubby toxins pod door toggle. Turns out the mass driver control `range` was a few tiles too short.

## Why It's Good For The Game
Var edits bad
Fixes: #54444

## Changelog
:cl: Denton
fix: Fixed the Pubby toxins mass driver door toggle button.
code: Replaced var edited mass driver machinery with proper subtypes.
/:cl:

### CODE REVIEW

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 1]
```diff
@@ -30959,6 +30959,7 @@
 	},
 /obj/machinery/computer/pod/old/mass_driver_controller{
 	id = "toxinsdriver";
+	range = 6;
```
Comment (Rohesie) [inline _maps/map_files/PubbyStation/PubbyStation.dmm:30962]: This kind of change should be done through a variant type IMO.
No other var-edited mass drivers have this, and this is less maintainable.

Comment (81Denton) [reply _maps/map_files/PubbyStation/PubbyStation.dmm:30962]: Hmm agreed

Comment (81Denton) [reply _maps/map_files/PubbyStation/PubbyStation.dmm:30962]: Actually, let me have a look at all mass drivers and see if I can't group them together better.

**Comment by 81Denton** [comment]
@Rohesie I replaced all var edited mass driver machinery with proper subtypes that get their IDs from defines.

**Review by Rohesie — APPROVED** [review]

**Pull request merged**